.gitlab-ci.yml

```diff
@@ -166,6 +166,20 @@ build:packages:cross:
   before_script:
     - *dind-login
   script:
+    - DOCKER_RETRIES=4
+    - DOCKER_RETRY_SLEEP_S=1
+    - DOCKER_RUNNING=false
+    - while [[ $DOCKER_RETRIES -gt 0 ]]; do
+    -  docker ps && DOCKER_RUNNING=true
+    -  if [[ "${DOCKER_RUNNING}" == "true" ]]; then
+    -   break
+    -  fi
+    -  sleep "${DOCKER_RETRY_SLEEP_S}"
+    -  let DOCKER_RETRIES--
+    - done
+    - if [[ "${DOCKER_RUNNING}" != "true" ]]; then
+    -  exit 192
+    - fi
```

Comment:
Please put this somewhere else and reuse it in the other jobs:
* build:orig:debian:bullseye:amd64
* build:packages:sim

Reply:
actually I started all this with adding to the QA meeting agenda, I want to put it in some type of docker:required:template or something or this sort. we can test it in here, and then create a follow up ticket. and I will adhere to your comment. thank you.

Reply:
tried my best in the separate commit. what do you think?
also: `build:packages:sim` is already covered I think via `build:packages:cross:` right?